app/models/target.rb

```diff
@@ -0,0 +1,18 @@
+# == Schema Information
+#
+# Table name: targets
+#
+#  id         :integer          not null, primary key
+#  title      :string
+#  lat        :decimal(, )
+#  lng        :decimal(, )
+#  radius     :decimal(, )
+#  topic_id   :integer
+#  created_at :datetime         not null
+#  updated_at :datetime         not null
+#
+
+class Target < ApplicationRecord
+  validates :topic, :title, :lat, :lng, :radius, presence: true
+  belongs_to :topic
```

Comment:
We use as a structure template for the models that associations go above validations.